.php_cs.dist

```diff
@@ -0,0 +1,61 @@
+<?php
+$finder = PhpCsFixer\Finder::create()
+    ->exclude([ 'Library' ])
```

Comment:
No, the cs fixer will only fetch php files, so we only need to exclude source code which does not belong to us (or the project)